components/Tag/Tag.jsx

```diff
@@ -127,7 +127,7 @@ const Tag = ({ children, text, onClose, ...rest }) => (
 
 Tag.propTypes = {
   bold: PropTypes.bool,
-  children: PropTypes.string,
+  children: PropTypes.node,
```

Comment:
can I use multiple children?. If you answer is "yes", maybe it needs to change to oneOf array, right? 🧑‍🏭